package.json

```diff
@@ -148,6 +148,8 @@
     "jquery": "^3.5.1",
     "jquery-ui": "^1.12.1",
     "jsonwebtoken": "^8.5.1",
+    "mobx": "^6.0.4",
+    "mobx-react": "^7.0.5",
```

Comment:
Given the recent removal of the `observer`, do we even need `mobx-react` at this point?

Reply:
Agreed.  Removed.